website/docs/glue-catalog.md

```diff
@@ -149,6 +150,47 @@ From your terminal, run the glue crawler.
 Once the crawler succeeds, you’ll be able to query this Iceberg table from Athena,
 EMR and/or Redshift query engines.
 
+
+#### Method 2: Using XTable APIs to sync with AWS Glue Data Catalog directly
+This applies for Iceberg target format only.
+
+**Pre-requisites:**
+* Download iceberg-aws-X.X.X.jar from the [Maven repository](https://mvnrepository.com/artifact/org.apache.iceberg/iceberg-aws)
+* Download bundle-X.X.X.jar from the [Maven repository](https://mvnrepository.com/artifact/software.amazon.awssdk/bundle)
```

Comment:
Download AWS Java SDK bundle-X.X.X.jar ?

Reply:
<img width="587" alt="image" src="https://github.com/user-attachments/assets/b96ce836-bd26-4055-9e2c-5d21c049ec82">
 
This is unclear from docs.